src/main/java/seedu/address/logic/commands/GroupCommand.java

```diff
@@ -104,5 +104,17 @@ public CommandResult execute(Model model) throws CommandException {
         }
         return new CommandResult(returnMessage + nameStringLists);
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj instanceof GroupCommand) {
+            GroupCommand testCommand = (GroupCommand) obj;
+            if (testCommand.getTagName().equals(this.tag)
```

Comment:
If I remember correctly, this module discourages the use of `this` in code, unlike cs2030s